project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -457,11 +457,15 @@ function c4m_user_og_theme() {
 }
 
 /**
- * Implements hook_c4m_cta_block_alter().
+ * Implements hook_c4m_cta_block().
  *
  * Set the CTA block for visitors.
  */
-function c4m_user_og_c4m_cta_block_alter(&$block) {
+function c4m_user_og_c4m_cta_block() {
+  if (!user_is_anonymous()) {
```

Comment:
@DavidHernandez 
this will cause a contradiction with:
```php
   if (!user_is_anonymous()) {
     global $user;
 
     if (og_get_membership('node', $group->nid, 'user', $user->uid)) {
       // Current user is already a member of this group.
       return;
     }
   }
```

which is down below (line 478).